src/cim/components/ClusterDeployment/ClusterDeploymentWizardNavigation.tsx

```diff
@@ -14,13 +15,13 @@ const ClusterDeploymentWizardNavigation: React.FC<{
   const { currentStepId, setCurrentStepId, clusterDeployment } = React.useContext(
     ClusterDeploymentWizardContext,
   );
-
+  const { t } = useTranslation();
   return (
     <WizardNav>
       <WizardNavItem key="installation-type" content="Installation type" step={0} isDisabled />
       <WizardNavItem
         key="cluster-details"
-        content={wizardStepNames['cluster-details']}
+        content={wizardStepNames(t)['cluster-details']}
```

Comment:
For cases like this, it would be better to extract the call to the function and reuse it in multiple places

eg.

```
const stepNames = wizardStepNames(t);
<WizardNavItem
        key="cluster-details"
        content={stepNames['cluster-details']}
<WizardNavItem
        key="cluster-details"
        content={stepNames['hosts-selection']}
...
```

etc